components/UI/modal/modal.module.scss

```diff
@@ -0,0 +1,28 @@
+  
+.Modal {
+  position: fixed;
+  z-index: 500;
```

Comment:
NIT: any specific reason for 500 as z-index

Reply:
Ah! 500 is not needed, 1 shoul just do the job. Thanks